client/cmd/dexcctl/sample-dexcctl.conf

```diff
@@ -7,13 +27,30 @@
 ; proxyuser=
 ; proxypass=
 
-; Username and password to authenticate connections to a Dexc RPC server
+; Username and password to authenticate connections to a Dexc RPC server.
 ; rpcuser=
 ; rpcpass=
 
-; RPC server to connect to
+; RPC server to connect to.
 ; rpcaddr=localhost:5757
 
-; RPC server certificate chain file for validation
+; RPC server certificate chain file for validation.
 ; rpccert=~/.dexc/rpc.cert
 
+; ------------------------------------------------------------------------------
+; General settings
+; ------------------------------------------------------------------------------
+
+; Display version information and exit.
+; version=true
+
+; List all of the supported commands and exit.
+; listcommands=true
+
+; Print json messages sent and received.
+; json=true
+
+; Password arguments to bypass stdin prompts.
+; Multiple values may be provided by providing multiple password arguments, each
+; on a separate line with each line starting with "passarg=".
+; passarg=
```

Comment:
This one is also not great for the config file, because what is appropriate changes from command to command and isn't something that can be configured globally.  This could be described with an example command however.  For instance, consider the following used to setup a simnet DCR wallet, where there are two `passarg`s, the first for the app password, and the second for the wallet's passphrase.

`./dexcctl --simnet -p a -p abc newwallet 42 ~/dextest/dcr/beta/beta.conf '{"account":"default"}'`

Each command has it's own set of required password arguments, and `newwallet` requires these two.  See the `promptPasswords` map in client/cmd/dexcctl/main.go.